builder/amazon/ebs/builder.go

```diff
@@ -53,6 +53,12 @@ type Config struct {
 	// engine](/docs/templates/engine.html), see [Build template
 	// data](#build-template-data) for more information.
 	VolumeRunTags awscommon.TagMap `mapstructure:"run_volume_tags"`
+	// If you set this flag, we'll add clauses to the
```

Comment:
Note I've only added this option for the ebs builder, because I don't think it's relevant to the other Amazon builders.